src/DIRAC/WorkloadManagementSystem/Service/SandboxStoreHandler.py

```diff
@@ -95,10 +95,11 @@ def _getFromClient(self, fileId, token, fileSize, fileHelper=None, data=""):
         gLogger.info("Upload requested", f"for {aHash} [{extension}]")
 
         credDict = self.getRemoteCredentials()
-        vo = Registry.getVOForGroup(credDict["group"])
+        if "VO" not in credDict:
+            credDict["VO"] = Registry.getVOForGroup(credDict["group"])
```

Comment:
Is mutating the result of `getRemoteCredentials` a good idea?

Reply:
Good point. I modified the code.